interfaces/sourcegen/sourcegen/preamble.txt

```diff
@@ -0,0 +1,5 @@
+This file is part of Cantera. See License.txt in the top-level directory or
```

Comment:
I don't think this needs to be its own file, having a string constant in `sourcegen/__init__.py` or something like that would be fine, which could still be returned by `get_preamble`.